front/components/sparkle/SidebarContext.tsx

```diff
@@ -1,13 +1,17 @@
-import React, { useState } from "react";
+import React, { useCallback, useState } from "react";
 
```

Comment:
do you know why it's in /sparkle/ ?

Reply:
Navigation is going to move to Sparkle very soon.